amp.php

```diff
@@ -16,7 +16,7 @@
 define( 'AMP__VERSION', '0.4.2' );
 
 require_once( AMP__DIR__ . '/back-compat/back-compat.php' );
-require_once( AMP__DIR__ . '/includes/amp-helper-functions.php' );
```

Comment:
This file will need to continue to be included for the foreseeable future so that the stubs are defined and available, since themes and plugins that use these functions are just going to assume they are available if the AMP plugin is active. In this PR these functions could get `_deprecated_function()` calls added to them, and then maybe a year later this `require_once` could be removed, with the actual file then removed a year after that.